### ISSUE TITLE
Important error in NVDA2020.4RC1: The "Copy portable configuration to current user account" checkbox is missing when installing a portable copy on a computer that has never installed NVDA

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#12071]
### Steps to reproduce:
1. Run the installed NVDA and create a portable version.
2. Check the "Copy current user configuration" option.
3. Uninstall the installed NVDA and make sure there is no nvda directory in %appdata% after uninstalling.
4. Run the portable copy created in (1) and select "Install NVDA" under the "Tools" menu.
PS: You can also omit (3) and install the created portable copy in Windows Sandbox.

**Special attention is needed to ensure that there is no NVDA directory in %appdata%**

### Actual behavior:
"Copy portable configuration to current user account" is missing.

### Expected behavior:
"Copy portable configuration to current user account" is displayed normally and can be checked.

#### NVDA installed/portable/running from source:
only portable version

#### NVDA version:
NVDA2020.4RC1, alpha-21753

#### Windows version:
Windows 10 20H2 (64-bit) build 19042.746

#### Name and version of other software in use when reproducing the issue:
none

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
alpha-21753

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
none

**Comment by ABuffEr** [comment]
Hi,
I confirm to have noticed this issue some alpha versions ago, thanks @cary-rowen for investigation and STR.

**Comment by leonardder** [comment]
Just to make sure, is this a regression in 2020.4?

**Comment by leonardder** [comment]
Related to https://github.com/nvaccess/nvda/pull/9679

**Comment by cary-rowen** [comment]
I am not sure if this is a regression of pr9679, this problem has not been reproduced on nvda2020.3.
@lukaszgo1  @leonardder @feerrenrut Can you comment on this?

**Comment by feerrenrut** [comment]
If the issue does not show up in 2020.3 then it is a regression, I haven't verified this. That said, given the late stage of discovery I don't think this is serious enough to delay the release.
